tests/storage/zfsvol/test_zfsvol_sr.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import time
+import pytest
+
+from lib.commands import SSHCommandFailed
+from lib.common import wait_for, vm_image
+from tests.storage import vdi_is_open
+
+# Requirements:
+# - one XCP-ng host >= 8.3 with an additional unused disk for the SR
+# - access to XCP-ng RPM repository from the host
+
+pytestmark = pytest.mark.usefixtures("host_at_least_8_3")
```

Comment:
Is assigning to the pytestmark variable useful? We never had to do so in all our other uses of `pytest.mark.usefixtures`.

Reply:
That's the official way to apply the feature to the whole module (it's not possible to use annotations for that).

Just tested against a 8.2 host, all tests properly skipped with:
```
SKIPPED [1] tests/storage/zfsvol/test_zfsvol_sr.py:22: This test requires an XCP-ng >= 8.3 host
```

Reply:
Right. I thought I had used fixtures applied to whole modules before in xcp-ng-tests, but apparently I didn't.